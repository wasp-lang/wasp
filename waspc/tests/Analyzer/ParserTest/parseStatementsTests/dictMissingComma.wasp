test Declaration {
  a: 1
  b: 2
}
