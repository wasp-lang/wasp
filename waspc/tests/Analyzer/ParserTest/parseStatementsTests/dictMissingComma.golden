Parse error at 3:3:
  Unexpected token: b
  Expected one of the following tokens instead: '}',','

Parse error at 3:4:
  Unexpected token: :
  Expected one of the following tokens instead: <identifier>

Parse error at 3:6:
  Unexpected token: 2
  Expected one of the following tokens instead: <identifier>,<eof>

Parse error at 4:1:
  Unexpected token: }
  Expected one of the following tokens instead: <eof>
